src/seq.rs

```diff
@@ -915,10 +915,6 @@ impl EventData for EvNote {
              EventType::Noteoff |
              EventType::Keypress)
     }
-    fn get_data(ev: &Event) -> Self {
```

Comment:
Perhaps better to switch the order of methods in the trait itself than to switch all the implementations?

Reply:
I honestly still don't get this part. Why is the order of methods important? Are they supposed to be in alphabetic order? Or can they be in any order as long as they follow what the trait declaration does? Is there a coding convention document you're following, or is it just that you think it looks tidier that way?

Reply:
The trait and the impl should have the same order but it can be what you want but the trait decide for the impl. Just say me what order you want i i will change it.